examples/tutorial/flaskr/auth.py

```diff
@@ -56,7 +56,7 @@ def register():
         elif db.execute(
             'SELECT id FROM user WHERE username = ?', (username,)
         ).fetchone() is not None:
-            error = 'User {} is already registered.'.format(username)
+            error = 'User {0} is already registered.'.format(username)
```

Comment:
since when do we care about python 2.6 again? ;)

Reply:
For at least a few more hours.